src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -59,7 +59,8 @@ public string ClientVersion
 
         public bool IsBrokerEnabled { get; internal set; }
 
-        public WindowsBrokerOptions WindowsBrokerOptions { get; set; }
+        // Legacy options for UWP. .NET broker options are in BrokerOptions
+        public WindowsBrokerOptions UwpBrokerOptions { get; set; } 

```

Comment:
TODO: try to get rid of it completely.

Reply:
Needed for legacy such as MsaPassthrough Header for UWP apps.